packages/quantic/force-app/main/default/lwc/quanticSearchBox/quanticSearchBox.js

```diff
@@ -61,13 +86,18 @@ export default class QuanticSearchBox extends LightningElement {
   renderedCallback() {
     initializeWithHeadless(this, this.engineId, this.initialize.bind(this));
     if (!this.input) {
-      // @ts-ignore
       this.input = this.template.querySelector('input');
     }
     if (!this.combobox) {
-      // @ts-ignore
       this.combobox = this.template.querySelector('.slds-combobox');
     }
+    if (this.withoutSubmitButton && !this.clearButton) {
+      this.clearButton = this.template.querySelector('.slds-button__icon');
+      this.clearButton.hidden = true;
```

Comment:
Why are you setting `hidden = true` in this specific case?

Reply:
forgot to remove it after I started using `slds-hidden` instead. I'll remove it now